src/ray/util/logging_utils.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2025 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains a few logging related util functions.
+
+#include "ray/util/log_redirection_options.h"
+
+namespace ray {
+
+// Util functions to redirect stdout / stderr stream.
```

Comment:
I'd name the file `stream_redirection_utils.h`

Reply:
updated